client/human/GGHumanClientApp.cpp

```diff
@@ -443,7 +433,7 @@ void GGHumanClientApp::StartServer() {
         throw LocalServerAlreadyRunningException();
     }
 
-    std::string SERVER_CLIENT_EXE = ServerClientExe();
+    std::string SERVER_CLIENT_EXE = GetOptionsDB().Get<std::string>("freeoriond.path");
```

Comment:
option should probably be something like `misc.server-local-binary.path`.

(I assume this can't ever be used to start a remote server)

Reply:
No, I suppose support of remote process execution via ssh or something is outside of FreeOrion scope.

Reply:
Renamed option.